pkg/plugins/manifest.go

```diff
@@ -80,8 +82,9 @@ func readPluginManifest(body []byte) (*pluginManifest, error) {
 	return manifest, nil
 }
 
-// GetPluginSignatureState returns the signature state for a plugin
-func GetPluginSignatureState(plugin *PluginBase) PluginSignature {
+// getPluginSignatureState returns the signature state for a plugin.
+func getPluginSignatureState(log log.Logger, plugin *PluginBase) PluginSignature {
+	log.Debug("Getting signature state of plugin...", "plugin", plugin.Id)
```

Comment:
why `...` ? :)

Reply:
@bergquist I can remove it, I just tend to use it in case of lengthy operations.